--- a/net/core/dev.c
+++ b/net/core/dev.c
@@ -12237,7 +12237,7 @@ static int __init net_dev_init(void)
 
 #ifdef CONFIG_SMP
 	/* Validate cacheline groups fit in specified sizes */
-	CACHELINE_ASSERT_GROUP_SIZE(struct net_device, net_device_read_tx, 160);
+	CACHELINE_ASSERT_GROUP_SIZE(struct net_device, net_device_read_tx, 200);
 	CACHELINE_ASSERT_GROUP_SIZE(struct net_device, net_device_write_tx, 64);
 	CACHELINE_ASSERT_GROUP_SIZE(struct net_device, net_device_queue, 128);
 #endif
